commit c8c587b854c9e85fc9ce58c8192d532205f0ee1f
Author: Tamar Christina <tamar.christina@arm.com>
Date:   Wed Feb 21 11:42:13 2024 +0000

    AArch64: skip modes_1.f90 [PR107071]
    
    This test has never worked on AArch64 since the day it was committed.  It has
    a number of issues that prevent it from working on AArch64:
    
    The testfailures seem to be known and triaged, so until that's fixed there's
    no point in running this test.
    
    gcc/testsuite/ChangeLog:
    
            PR fortran/107071
            * gfortran.dg/ieee/modes_1.f90: skip aarch64, arm.

diff --git a/gcc/testsuite/gfortran.dg/ieee/modes_1.f90 b/gcc/testsuite/gfortran.dg/ieee/modes_1.f90
index 205c47f3800..e29d8c678e6 100644
--- a/gcc/testsuite/gfortran.dg/ieee/modes_1.f90
+++ b/gcc/testsuite/gfortran.dg/ieee/modes_1.f90
@@ -1,5 +1,5 @@
 ! { dg-do run }
-!
+! { dg-skip-if "PR libfortran/78314" { aarch64*-*-gnu* arm*-*-gnueabi arm*-*-gnueabihf } }
 ! Test IEEE_MODES_TYPE, IEEE_GET_MODES and IEEE_SET_MODES
 
 
